src/entities/amm/amm.ts

```diff
@@ -1482,6 +1483,36 @@ export class AMM {
     return tickToFixedRate(closestUsableTick).toNumber();
   }
 
+  // historocal rates
+
+  public async getHistoricalVariableRate(
+    subgraphUrl: string,
```

Comment:
replace with `chainId` and use `getSubgraphURL(chainId, SubgraphURLEnum.voltzProtocol)` to get the subgraph URL

Reply:
ah nicee. I implemented that [13c3037](https://github.com/Voltz-Protocol/v1-sdk/pull/206/commits/13c30377bd06863b9fa0ffd0afeae77c4d4abda3)